### ISSUE TITLE
Process JSON?

### ISSUE THREAD

**Issue by eklem** [fergiemcdowall/norch-document-processor#6]
We should have a way of process JSON in addition to HTML since a lot of services offer JSON straight out of the box.

Use case for this is that I'm trying to index my online life and make it searchable. I got the idea on one way to solve it from this blog: http://codepen.io/nickmoreton/blog/using-ifttt-and-google-drive-to-create-a-json-api That's giving me JSON directly from Google Spreadsheet, but I still need to remove a lot of noise: http://jsonformatter.curiousconcept.com/#https://spreadsheets.google.com/feeds/list/1IBZJCwh3oErZog8rqWLQm6cxSooCGAdTfNpzHvkUvUw/od6/public/values?alt=json

Basically, I want to remove everything before "entry" and massage/adjust in the rest of the file.

**Comment by eklem** [comment]
And with [Kimono Labs](http://kimonolabs.com) this is an even better use case. Now it is really easy to get okay JSON, and then you only need to massage it a bit to get it in shape for `Norch-document-processor`.
